test-app/gradle.properties

```diff
@@ -0,0 +1,16 @@
+mlHost=localhost
+# Define these in gradle-local.properties
+mlUsername=
```

Comment:
These can both be admin/admin now right, since we're assuming the use of Docker?